book/MovingToTheGPU.html

```diff
@@ -10,52 +10,52 @@
 
 Introduction
 ====================================================================================================
-_Ray Tracing_ is a rendering method that simulates the flow of light. It can faithfully recreate a
-variety of optical phenomena and can be used to render photorealistic images. It is one of the
-foundational techniques in Computer Graphics used to model light transport.
+_Ray Tracing_ is a rendering method in Computer Graphics that simulates the flow of light.
+It can accurately recreate a variety of optical phenomena and be used to render photorealistic images.
 
-_Path tracing_ is an application of this approach in order to compute _Global Illumination_. Its
+_Path tracing_ is an application of this approach, which is used to compute _Global Illumination_. Its
 core idea is to repeatedly trace millions of random rays through the scene and bounce them off
-objects based on surface properties. The algorithm is remarkably simple and it is relatively easy
-to build a basic path tracer that supports a small number of material and geometry types. Peter
-Shirley's [_Ray Tracing In One Weekend_][RTIOW] shows you how to get started with a CPU path tracer
+objects based on surface properties. The algorithm is remarkably simple and relatively easy
+to build a basic path tracer, which supports a small number of material and geometry types. Peter
+Shirley's [_Ray Tracing In One Weekend_][RTIOW] (RTIOW) shows you how to get started with a CPU path tracer
```

Comment:
The acronym "RTIOW" is used without being introduced.

Reply:
In Markdeep, the text inside the square brackets is the link text. Here, "_Ray Tracing in One Weekend_".